### PR TITLE
Add the UI to the webpage

### PR DESCRIPTION
**Pull request by kelbrown20**
**Description:**  Add a link to to the UI to the website landing page

### CODE REVIEW

File: src/pages/collaborate.tsx [hunk 1]
```diff
@@ -12,4 +12,4 @@ const Collaborate = () => {
   return null;
 };
 
-export default Collaborate;
\ No newline at end of file
+export default Collaborate;
```
Comment (kelbrown20) [inline src/pages/collaborate.tsx:15]: I was running into an error that told me to add a space here. So thats what this is for 😄

**Review by vishnoianil — COMMENTED** [review]
@kelbrown20 awesome!! Looks good to me.

**Comment by kelbrown20** [comment]
Can @instructlab/website-maintainers PTAL when you get the chance to confirm this? Thank you 🙂

**Review by RobotSail — APPROVED** [review]
LGTM

**Comment by kelbrown20** [comment]
@cybette Just updated that! Should be all set now

**Review by cybette — APPROVED** [review]
Brilliant, thank you so much @kelbrown20!

**Comment by RobotSail** [comment]
Thank you for adding this @kelbrown20 !!

**Pull request merged**